src/main/webapp/js/instructorFeedbackResults.js

```diff
@@ -209,14 +209,21 @@ function toggleCollapse(e, pans) {
     var panels = pans || $('div.panel-collapse');
 
     if ($(e).html().trim().startsWith(expand)) {
-        isExpandingAll = true;
-        var i = 0;
-        for (var idx = 0; idx < panels.length; idx++) {
-            if ($(panels[idx]).attr('class').indexOf('in') === -1) {
-                setTimeout(showSingleCollapse, 50 * i, panels[idx]);
-                i++;
+        $('.ajax-response-auto').click();
+        var $autoLoadCount = $('.ajax_auto').click();
+        var $isAutoLoadingTriggered = $autoLoadCount.length !== 0;
+
+        if (!$isAutoLoadingTriggered) {
+            isExpandingAll = true;
+            var i = 0;
+            for (var idx = 0; idx < panels.length; idx++) {
+                if ($(panels[idx]).attr('class').indexOf('in') === -1) {
+                    setTimeout(showSingleCollapse, 50 * i, panels[idx]);
```

Comment:
Yes, leave it unchanged for now, @li-kai has opened an issue regarding it and we have collectively added some good readings for whoever wants to take up refactoring it in the future